test/unittests/execute_call_test.cpp

```diff
@@ -210,35 +203,43 @@ TEST(execute_call, call_indirect_uninited_table)
 
 TEST(execute_call, imported_function_call)
 {
-    Module module;
-    module.typesec.emplace_back(FuncType{{}, {ValType::i32}});
-    module.importsec.emplace_back(Import{"mod", "foo", ExternalKind::Function, {0}});
-    module.funcsec.emplace_back(TypeIdx{0});
-    module.codesec.emplace_back(Code{0, {Instr::call, Instr::end}, {0, 0, 0, 0}});
+    /* wat2wasm
+    (import "mod" "foo" (func (result i32)))
+    (func (result i32)
+      call 0
+    )
+    */
+    const auto wasm = from_hex(
+        "0061736d010000000105016000017f020b01036d6f6403666f6f0000030201000a0601040010000b");
+
+    const auto module = parse(wasm);
 
-    auto host_foo = [](Instance&, std::vector<uint64_t>) -> execution_result {
+    constexpr auto host_foo = [](Instance&, std::vector<uint64_t>) -> execution_result {
```

Comment:
Does the `constexpr` matters here?

Reply:
Not very much. This follows the fashion to `const` everything possible. Here it also means that this lambda is compile-time constant and does not depend on variables around it.